xarray/structure/alignment.py

```diff
@@ -216,30 +216,37 @@ def _normalize_indexes(
 
         normalized_indexes = {}
```

Comment:
Can we add a comment describing what "normalize" means here please?

Reply:
That was confusing indeed. In [299de61](https://github.com/pydata/xarray/pull/10293/commits/299de61a694e1f8ba92fd32d2fbe67f16b1369f9) I split the logic in two separate function/method. It is much clearer now, hopefully.